python/tests/res/simulator/test_batch_sim.py

```diff
@@ -405,7 +401,11 @@ def test_stop_sim(self):
                                   })
                              ])
 
+            time.sleep(2.0)
             ctx.stop()
+            wait_until(
```

Comment:
Why is this necessary?

Reply:
`context.stop()` is blocking operation now, but needs to be tested a few times first - then it's gone